swupd/manifest.go

```diff
@@ -598,6 +598,7 @@ func addAllManifestFiles(manifests []*Manifest, ui UpdateInfo, c config, numWork
 		defer wg.Done()
 		for m := range mCh {
 			if err := m.addManifestFiles(ui, c); err != nil {
+				fmt.Println(err)
```

Comment:
What was the reason for adding `fmt.Println(err)` in several locations in this PR? Since we are returning an error, I would expect the error message to eventually be printed again.

Reply:
I added it for debugging purpose. We are only catching the first failure. As we are running multiple go routines, logging the errors in other go routines.